packages/neon_framework/lib/src/utils/push_utils.dart

```diff
@@ -185,6 +185,29 @@ class PushUtils {
             ),
             payload: json.encode(pushNotification.toJson()),
           );
+
+          // Other platforms don't support grouping, so don't send another notification there.
+          if (defaultTargetPlatform == TargetPlatform.android) {
+            // Sine we only support Android SDK 24+, we can just generate an empty summary notification as it will not be shown anyway.
```

Comment:
```suggestion
            // Since we only support Android SDK 24+, we can just generate an empty summary notification as it will not be shown anyway.
```